x-pack/plugins/security_solution/public/detections/configurations/security_solution_detections/render_cell_value.tsx

```diff
@@ -61,18 +55,19 @@ export const RenderCellValue: React.FC<EuiDataGridCellValueElementProps & CellVa
   );
 
   return columnId === SIGNAL_RULE_NAME_FIELD_NAME &&
-    suppressionCount?.value &&
-    parseInt(suppressionCount.value[0], 10) > 0 ? (
-    <SuppressedAlertIconWrapper>
-      <EuiToolTip
-        position="top"
-        content={SUPPRESSED_ALERT_TOOLTIP(parseInt(suppressionCount.value[0], 10))}
-      >
-        <EuiIcon type="layers" />
-      </EuiToolTip>
-      &nbsp;
-      {component}
-    </SuppressedAlertIconWrapper>
+    suppressionCount &&
+    parseInt(suppressionCount[0], 10) > 0 ? (
+    <EuiFlexGroup gutterSize="xs">
+      <EuiFlexItem grow={false}>
+        <EuiToolTip
+          position="top"
```

Comment:
style nit: The icon is up against the edge of the popover as all the other buttons in that popover have an `8px` padding on the edge

Reply:
Hmm so I tried adding the padding here and it fixes the popover but then adds it to the rule name in the table row as well, which isn't desirable. Any idea why wrapping the component with a flex group like this causes the padding not to apply in the popover anymore? Field values without the new layers icon look like they get the 8px padding automatically.

Reply:
Not sure, I think that's might be an EUI quirk?